duplicate-check.user.js

```diff
@@ -0,0 +1,86 @@
+// ==UserScript==
+// @name         Open Candidate Profile in new Tab if Duplicate Badge is detected
+// @namespace    https://canonical.com/
+// @version      2024-11-25
```

Comment:
Small change, just to keep this versioning in line with the other scripts. Otherwise it all looks good!

```suggestion
// @version      0.1.0
```